backend-plugin/src/main/java/com/redhat/jenkins/nodesharingbackend/Pool.java

```diff
@@ -284,4 +289,58 @@ public void generateResponse(StaplerRequest req, StaplerResponse rsp, Object nod
             rsp.sendError(HttpServletResponse.SC_NOT_IMPLEMENTED, getMessage());
         }
     }
+
+    // Helpers for JCasC
+    @CheckForNull
+    public String getConfigRepo() {
+        return System.getProperty(CONFIG_REPO_PROPERTY_NAME);
+    }
+
+    @DataBoundSetter
+    public void setConfigRepo(@Nonnull final String configRepo) {
+        String prop = getConfigRepo();
+        if (prop != null && !Objects.equals(prop, configRepo)) {
```

Comment:
Should have been more clear, with the `Object.equals` there is no need for the null check.

Reply:
I thought the same idea, but in pure JCasC orchestration, the `prop` is NULL and `configRepo` contains the real value from YAML so `Objects.equas(...)` returns FALSE and then it is detected as a clashing (wrong!)...

Reply:
Oh, my mistake. You are correct...